shared/src/business/entities/DocketEntry.ts

```diff
@@ -466,15 +480,194 @@ export class DocketEntry extends JoiValidationEntity {
   }
 
   isCourtIssued(): boolean {
-    return DocketEntry.isCourtIssued(this.eventCode);
+    return DocketEntry.isCourtIssued({ eventCode: this.eventCode });
   }
 
-  static isCourtIssued(eventCode: string): boolean {
+  static TRANSCRIPT_AGE_DAYS_MIN = 90;
+
+  static isCourtIssued({ eventCode }: { eventCode: string }): boolean {
     return COURT_ISSUED_EVENT_CODES.map(
       ({ eventCode: courtIssuedEventCode }) => courtIssuedEventCode,
     ).includes(eventCode);
   }
 
+  static isAvailableToExternal = (entry: RawDocketEntry): boolean => {
+    const servedOrUnservable =
+      DocketEntry.isServed(entry) || DocketEntry.isUnservable(entry);
+
+    return (
+      servedOrUnservable &&
```

Comment:
What about if the docket entry has been sealed?

Reply:
Sealed Docket entries are technically available to some External users.